waku/v2/utils/metrics_logging.nim

```diff
@@ -0,0 +1,79 @@
+{.push raises: [Defect].}
+
+import
+  stew/results,
+  stew/shims/net,
+  chronicles,
+  chronos,
+  metrics,
+  metrics/chronos_httpserver
+import
+  ../protocol/waku_filter,
+  ../protocol/waku_store,
+  ../protocol/waku_lightpush,
+  ../protocol/waku_swap/waku_swap,
+  ../protocol/waku_peer_exchange,
+  ../utils/collector,
+  ../node/peer_manager/peer_manager,
+  ../node/waku_node
+
+when defined(rln) or defined(rlnzerokit):
+  import ../protocol/waku_rln_relay/waku_rln_relay_metrics
+
+const LogInterval = 30.seconds
+
+logScope:
+  topics = "wakunode.setup.metrics"
```

Comment:
Now rather something like `waku.utils.metrics`

Reply:
Addressed in [9f2d9a9](https://github.com/status-im/nwaku/pull/1279/commits/9f2d9a92de06f2f740f3c4722b5a6a797a89a089)